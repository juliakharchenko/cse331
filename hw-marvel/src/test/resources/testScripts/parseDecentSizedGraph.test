# Test the parser on a mid sized graph of nba players as nodes
# and multiple edges per node.

LoadGraph nba nbaPlayers.csv
ListChildren nba LEBRON-JAMES
ListChildren nba MICHAEL-JORDAN
ListChildren nba MAGIC-JOHNSON
ListChildren nba KOBE-BRYANT
ListChildren nba KAREEM-ABDUL-JABBAR
ListChildren nba DWYANE-WADE
ListChildren nba REGGIE-MILLER