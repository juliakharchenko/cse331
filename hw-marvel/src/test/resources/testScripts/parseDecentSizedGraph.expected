# Test the parser on a mid sized graph of nba players as nodes
# and multiple edges per node.

loaded graph nba
the children of LEBRON-JAMES in nba are: DWYANE-WADE(CAVALIERS) DWYANE-WADE(HEAT) KAREEM-ABDUL-JABBAR(LAKERS) KOBE-BRYANT(LAKERS) MAGIC-JOHNSON(LAKERS)
the children of MICHAEL-JORDAN in nba are: DWYANE-WADE(BULLS)
the children of MAGIC-JOHNSON in nba are: KAREEM-ABDUL-JABBAR(LAKERS) KOBE-BRYANT(LAKERS) LEBRON-JAMES(LAKERS)
the children of KOBE-BRYANT in nba are: KAREEM-ABDUL-JABBAR(LAKERS) LEBRON-JAMES(LAKERS) MAGIC-JOHNSON(LAKERS)
the children of KAREEM-ABDUL-JABBAR in nba are: KOBE-BRYANT(LAKERS) LEBRON-JAMES(LAKERS) MAGIC-JOHNSON(LAKERS)
the children of DWYANE-WADE in nba are: LEBRON-JAMES(CAVALIERS) LEBRON-JAMES(HEAT) MICHAEL-JORDAN(BULLS)
the children of REGGIE-MILLER in nba are: